index.html

```diff
@@ -1,19 +1,106 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <!--body starts here-->
+  <body>
+    <!--header starts here-->
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="karma-logo" class="navbar-logo" />
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+      <nav>
+        <ul class="navbar-list">
+          <li id="navbar-button"><a href="#">Meet Karma</a></li>
+          <li id="navbar-button"><a href="#">How It Works</a></li>
+          <li id="navbar-button"><a href="#">Store</a></li>
+          <li id="navbar-button"><a href="#">Blog</a></li>
+          <li id="navbar-button"><a href="#">Help</a></li>
+          <li id="navbar-button"><a href="#">Login</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!--main starts here-->
+    <main>
+      <section class="main-articles">
+        <article class="article-one">
+          <img
+            src="img/first-background.jpg"
+            alt="first-background"
+            class="background-image"
+          />
+          <h1 class="line-one">Introducing Karma</h1>
+          <h2 class="line-two">Bring WiFi with you, everywhere you go.</h2>
+          <button class="button">Learn More</button>
+        </article>
 
-</body>
+        <article class="article-two">
+          <h3 class="little-karma">Everyone Needs a Little Karma.</h3>
+          <ul class="icones-list">
+            <li class="photo-text">
+              <img
+                src="img/icon-devices.svg"
+                alt="devices-icon"
```

Comment:
Alt text is to be read out eg by a screenreader, so should be a normal sentence that describes the image eg 'Picture of some devices'